pkg/lib/operation/project/remote/table/preview/operation.go

```diff
@@ -77,7 +77,7 @@ func Run(ctx context.Context, o Options, d dependencies) (err error) {
 }
 
 func getPreviewOptions(o *Options) []keboola.PreviewOption {
-	var opts []keboola.PreviewOption
+	opts := make([]keboola.PreviewOption, 0, 4+len(o.WhereFilters)+len(o.Order))
 	if o.Limit > 0 {
 		opts = append(opts, keboola.WithLimitRows(o.Limit))
```

Comment:
`4+len(o.WhereFilters)+len(o.Order)` ?

Are you sure you want to have this here, what is the benefit?

Reply:
You have 4 options, then filters and then order. If you pre alocate the slice for expected amount of entries, it does not need to realocate and it is faster. Ofc this is just optimization of CLI, does not have very huge effect but also why don't you allocate exact number of the entries that you are expected to insert using `append` ?